app/src/ui/lib/list/list-row.tsx

```diff
@@ -107,13 +107,13 @@ interface IListRowProps {
 
   /** Optional role setting.
    *
-   * By default our lists use the `list-box` role paired with list items of role
+   * By default our lists use the `listbox` role paired with list items of role
    * 'option' because that have selection capability. In that case, a
    * screenreader will only browse to the selected list option. If the list is
    * meant to be informational as opposed for selection, we should use `list`
    * with `listitem` as the role for the items so browse mode can navigate them.
    */
-  readonly role?: `option` | `listitem` | 'presentation'
```

Comment:
I also for some reason used backtick.. when it would more properly be a single quote... surprised the linter didn't hammer me on that.